src/styles/card.scss

```diff
@@ -0,0 +1,128 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+main {
+  display: grid;
+  padding: 50px 40px;
+  justify-content: center;
+  gap: 46px 48px;
+  grid-template-columns: repeat(1, 200px);
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  transition: transform 300ms ease;
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $card-dark-text;
+    transition: color 300ms ease;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-grey-text;
+    align-self: flex-start;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__name-price {
+    font-size: 12px;
+    color: $card-grey-text;
+  }
+
+  &__num-price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $card-dark-text;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    font-family: Roboto, sans-serif;
+    color: #fff;
+    width: 100%;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    background-color: $header-blue-color;
+    border-radius: 5px;
+    border: 1px solid $header-blue-color;
+    outline: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    text-align: center;
+    transition:
+      background-color 300ms ease,
+      color 300ms ease;
+    &:hover {
+      background-color: #fff;
+      border: 1px solid $header-blue-color;
+      color: $header-blue-color;
```

Comment:
The variable `$header-blue-color` is used for the color property in the hover state, but it's not defined in this file. Ensure that this variable is defined in a file that is imported into this SCSS file, or define it here to avoid compilation errors.